base/tests/integrationtests/com/google/idea/blaze/base/sync/ImportRootsTest.java

```diff
@@ -235,39 +235,15 @@ public void testContainsWorkspacePath_excludedParentsAreHandled() throws Excepti
   }
 
   @Test
-  public void testBazelIgnoreParser_worksWithoutBazelIgnore() throws Exception {
-    BazelIgnoreParser parser = new BazelIgnoreParser(workspaceRoot);
-    assertThat(parser.getIgnoredPaths()).isEmpty();
-  }
-
-  @Test
-  public void testBazelIgnoreParser_worksWithEmptyBazelIgnore() throws Exception {
-    workspace.createFile(new WorkspacePath(".bazelignore"), "");
-    BazelIgnoreParser parser = new BazelIgnoreParser(workspaceRoot);
-    assertThat(parser.getIgnoredPaths()).isEmpty();
-  }
-
-  @Test
-  public void testBazelIgnoreParser_doesNotCrashWithMalformedBazelIgnore() throws Exception {
-    // directories cannot be absolute and cannot contain ":"
-    workspace.createFile(new WorkspacePath(".bazelignore"), "/mal:formed");
-    BazelIgnoreParser parser = new BazelIgnoreParser(workspaceRoot);
-    assertThat(parser.getIgnoredPaths()).isEmpty();
-  }
+  public void testIgnoresPathsFromBazelIgnore() throws Exception {
+    workspace.createFile(new WorkspacePath(".bazelignore"), "root1/a/b");
 
-  @Test
-  public void testBazelIgnoreParser_excludesIgnoredPath() throws Exception {
-    workspace.createFile(new WorkspacePath(".bazelignore"), "root0");
-    BazelIgnoreParser parser = new BazelIgnoreParser(workspaceRoot);
-    assertThat(parser.getIgnoredPaths()).containsExactly(new WorkspacePath("root0"));
-  }
+    ImportRoots importRoots =
+        ImportRoots.builder(workspaceRoot, BuildSystem.Blaze)
+            .add(DirectoryEntry.include(new WorkspacePath("root0")))
+            .add(DirectoryEntry.include(new WorkspacePath("root1/a/b/c")))
+            .build();
```

Comment:
Move to BazelIgnoreParserTest?

Reply:
I left this test here because it tests how .bazelignore integrates with ImportRoots. I am adding another test to show that it only works with Bazel buildsystem. Do you think these tests belong in BazelIgnoreParserTest? Let me know if I should move them.

Reply:
Can you restore all the tests into this file then? No need to split the tests into two files, and keeps the git blame clean.